devlab/jenkins/run_functional_tests.sh

```diff
@@ -2,36 +2,31 @@
 set -e
 set -x
 
-WORKSPACE="${WORKSPACE:-$( cd $( dirname "$0" ) && cd ../../../ && pwd)}"
-
-CF_DIR=$WORKSPACE/cloudferry
-JOB_NAME="${JOB_NAME:-cloudferry-functional-tests}"
-
-trap 'clean_exit $LINENO $BASH_COMMAND; exit' SIGHUP SIGINT SIGQUIT SIGTERM EXIT
-clean_exit()
-{
-    pushd ${CF_DIR}/devlab
-    vboxmanage list vms
-    vagrant destroy --force
-    popd
-}
+export WORKSPACE="${WORKSPACE:-$( cd $( dirname "$0" ) && cd ../../../ && pwd)}"
+export CF_DIR=$WORKSPACE/cloudferry
+export JOB_NAME="${JOB_NAME:-cloudferry-functional-tests}"
+
+#trap 'clean_exit $LINENO $BASH_COMMAND; exit' SIGHUP SIGINT SIGQUIT SIGTERM EXIT
+#clean_exit()
+#{
+#    pushd ${CF_DIR}/devlab
+#    vboxmanage list vms
+#    vagrant destroy --force
+#    popd
+#}
```

Comment:
Is it a debug leftover? Keeping commented code is a bad practice, also I believe script should handle any error properly, which means we should keep this code uncommented.